tests/functional/firefox/new/test_download.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.new.download import DownloadPage
+
+
+# We don't currently show the download button for up-to-date Firefox users.
+# TODO explore how could we run this test using a known out-of-date Firefox?
+@pytest.mark.skip_if_firefox
+@pytest.mark.smoke
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_download_button_displayed(base_url, selenium):
+    page = DownloadPage(base_url, selenium).open()
+    assert page.download_button.is_displayed
+
+
+@pytest.mark.skip_if_firefox
+@pytest.mark.skip_if_internet_explorer(reason='https://github.com/SeleniumHQ/selenium/issues/448')
```

Comment:
reason is a nice touch, we should check for it and include it in the skip message in https://github.com/mozilla/bedrock/blob/master/tests/functional/conftest.py#L28-L35